### DISCUSSION TITLE
Run previous failed test files first

### DISCUSSION THREAD

**Discussion by manuelmeister** [sorry-cypress/sorry-cypress#507] [Ideas]
It would not skip any tests (as they may fail due to the changes), but would more quickly show if the previous failed tests are now resolved.

**Comment by agoldis** [comment]
Good idea / request, possible implementation would be:
- store previous spec files results in an effective data structure on MongoDB (e.g. fail rate, last run result)
- create a "lookup" hash that, given a spec file and projectId, find run order
- implement different strategy for requesting the next task - simple, fails based etc.
- Configure at project level
